index.js

```diff
@@ -100,23 +147,27 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
+const triple = (num => num * 3)
+
+console.log(myMap([1,2,3,4,5], triple)) 
 // // => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
 
-// console.log(myMap(["Raffy", "Chase"], greet)) 
+const greet = name => `Hi, ${name}!`
+
+console.log(myMap(["Raffy", "Chase"], greet)) 
```

Comment:
Nice work using arrow functions to make your code extremely concise. Great use of the one place JS provides implicit return values (in a single like no bracket arrow function). 

Only comment would be that you wrote your arrow functions using slightly different syntax. Always try to keep some consistency when writing your code. You should be able to drop the parens on your triple function.

const triple = num => num * 3
const greet = name => `Hi, ${name}!`

Reply:
Is either way (with or without parentheses) generally considered "more proper" that that I can practice developing that method?